sql/core/src/test/resources/sql-tests/results/cte-nonlegacy.sql.out

```diff
@@ -166,3 +166,16 @@ SELECT (
 struct<scalarsubquery():int>
 -- !query output
 3
+
+
+-- !query
+WITH t(c) AS (SELECT 1)
+SELECT * FROM t
+WHERE c IN (
+  WITH t(c) AS (SELECT 2)
+  SELECT * FROM t
+)
+-- !query schema
+struct<c:int>
+-- !query output
+
```

Comment:
@peter-toth . The result looks wrong because it's empty. We are expecting `1`, aren't we?